src/App/Components/Home/Home.js

```diff
@@ -0,0 +1,43 @@
+import React, { Component } from 'react';
+import axiosInstance from '../../../../configs/axios';
+export class Home extends Component {
+  state = {
+    up: false,
+    down: false
+  }
+  async componentDidMount() {
+    const res = await axiosInstance.get('');
+    if (res) this.setState({ up: true })
+    else this.setState({ down: true })
+  }
+  render() {
+    if (this.state.up) {
+      return (
+        <React.Fragment>
```

Comment:
@nkpremices any specific reason for using `React.Fragment` here instead of a div?

Reply:
Right now it's because all those components that I created are just dummy components @MemunaHaruna. But later on, when I will need a component (for e.g `Home`)to return multiple components, react fragment is a good choice.